src/Sender.php

```diff
@@ -164,7 +164,7 @@ public function sendSticker($to, $stickerId, $stickerPackageId, $stickerVersion)
      * @param array $content
      * @return $this|mixed
      */
-    public function sendMessage($to, $content)
+    protected function sendMessage($to, $content)
```

Comment:
__[phpmd]__
* Avoid variables with short names like $to. Configured minimum length is 3.